app/Files/TemporaryFile.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
why don't just directly to the folder?
1. upload file into folder. Can still process
2. User cancel upload -> delete file directly
3. User click submit -> OK